typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -26,11 +31,84 @@ import mantaTIAAddresses from './warp/manta-TIA-addresses.json';
 
 const repo = 'gcr.io/abacus-labs-dev/hyperlane-agent';
 
+// The chains here must be consistent with the environment's supportedChainNames, which is
+// checked / enforced at runtime & in the CI pipeline.
+//
+// This is intentionally separate and not derived from the environment's supportedChainNames
+// to allow for more fine-grained control over which chains are enabled for each agent role.
+export const hyperlaneContextAgentChainConfig: AgentChainConfig = {
+  [Role.Validator]: {
+    [Chains.arbitrum]: true,
```

Comment:
I expect this is maybe controversial - it's verbose but I think forcing us to be explicit, including when we *don't* want to support a chain, will help

Reply:
Definitely agree that moving from typescript to JSON / YAML would help in the move to make configs more explicit - this is my agenda for when we move more things out into the CLI. I've been having some conversations recently around this and think we'll start making moves toward this soon

Until we build out a better setup for ensuring "correctness" of parsed JSON/YAML configs, I think there are some benefits to keeping it in TS, like being able to use enums & keeping all agent deployment configuration within a single file